tests/functional/requirements.txt

```diff
@@ -1,2 +1,2 @@
-git+https://github.com/openstack-charmers/zaza.git@libjuju-3.1#egg=zaza
```

Comment:
To avoid breaking changes in master, I think it would be better to add `@<commit hash>` of the current master.

Reply:
I think it will be better that we change this line after https://github.com/openstack-charmers/zaza/pull/608 get merged.